### PR TITLE
Landing header

### PR DESCRIPTION
**Pull request by JSpring0303**

### CODE REVIEW

File: src/routes/(app)/app/account/create_key_dialog.svelte [hunk 1]
```diff
@@ -7,9 +7,16 @@
...
+<<<<<<< HEAD
+<<<<<<< HEAD
+	import { Button } from "@/ui/button";
+=======
+=======
 
+>>>>>>> 46ed1164eb2a5d4f64630e0c44a5619ba80e742b
 	import { Button, buttonVariants } from "@/ui/button";
 	import InlineCodeBlock from "@/inline-codeblock.svelte";
+>>>>>>> cd6736bb39d0423671ae1834afa4f5a47e45f3a7
```
Comment (d3rpp) [inline src/routes/(app)/app/account/create_key_dialog.svelte]: My man this is what a merge conflict looks like

File: src/routes/(landing)/+layout.svelte [hunk 2]
```diff
@@ -9,8 +9,9 @@
 
 <LandingHeader {user} />
 
-<Main class="h-landing-main">
-	{@render children()}
-</Main>
-
+<div class="bg-cover bg-center h-full" style="background-image: url('/src/lib/backgroundsAndAssets/landerDecoBackgroundV2.svg'), linear-gradient(to right, #000000 30%, #6CFF96 100%); background-position: right center; background-repeat: no-repeat; background-size: 40%;">
```
Comment (d3rpp) [inline src/routes/(landing)/+layout.svelte]: this won't work in a build, you'll need to add the asset to the `static` folder and reference it from there.

It works in dev, but not in a build

File: src/routes/(landing)/+page.svelte [hunk 3]
```diff
@@ -3,39 +3,37 @@
...
-		{:else if $query.isError}
-			<pre>{JSON.stringify($query.error, null, 2)}</pre>
-		{:else}
-			<pre>{JSON.stringify($query.data, null, 2)}</pre>
-		{/if}
+	<div class="flex items-center justify-between w-full gap-x-24 pt-32">
+
+		<h1 
+		class="flex-1 text-left" 
+		style="font-family: futura, sans-serif; font-weight: 400; font-style: normal; font-size: 4.75rem; text-align: left; color: white; text-shadow: -4px 5px 7px rgba(0, 0, 0, 0.4);">
```
Comment (d3rpp) [inline src/routes/(landing)/+page.svelte]: why we using inline styles?

**Review by d3rpp — CHANGES_REQUESTED** [review]
my man

File: src/routes/(landing)/+layout.svelte (hunk 2 above)
Comment (JSpring0303) [reply src/routes/(landing)/+layout.svelte]: The only problem is that it keeps going on about a vite allow list when i keep it in static

File: src/routes/(landing)/+page.svelte (hunk 3 above)
Comment (JSpring0303) [reply src/routes/(landing)/+page.svelte]: habit

**Review by d3rpp — APPROVED** [review]
LGTM :+1:

**Pull request merged**